Knittery/Details/PatternDetailsViewModel.swift

```diff
@@ -60,11 +60,11 @@ class PatternDetailsViewModel: ObservableObject {
     
     func getDownloadLinks() {
         if let downloadLocation = pattern.downloadLocation {
```

Comment:
The `if let` here could be changed to a `guard let` to get rid of one layer of curly brackets

Reply:
Done, good catch